<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BHARATH" failures="1" tests="3" name="Test" time="22.475" errors="0" timestamp="2021-05-11T08:40:08 IST">
  <testcase classname="parallPack.ParallelExecution" name="test" time="10.156"/>
  <testcase classname="parallPack.ParallelExecution" name="testA" time="4.831">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at parallPack.ParallelExecution.testA(ParallelExecution.java:54)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testA -->
  <testcase classname="parallPack.ParallelExecution" name="testB" time="7.315">
    <skipped/>
  </testcase> <!-- testB -->
</testsuite> <!-- Test -->
